core/commands/filestore.go

```diff
@@ -98,10 +98,11 @@ The output is:
 					errors bool
 				)
 
-				for {
+				for cmds.HandleError(err, res, re) {
 					v, err = res.Next()
 					if err != nil {
-						break
+						continue
+						// HandleError will return break and forward the error
```

Comment:
This wasn't how I intended `cmds.HandleError` to be used.  I think
```
for {
  v, err = res.Next()
  if !cmds.HandleError(err, res, re) {
    break
  }
```
would be better